bb-practice.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Blackberry Page</title>
+        <style>
+            .container{
+                max-width: 1200px;            
+            }
+            .row {
+                clear: both;
+            }
+            .col {
+                width: 25%;
+                float: left;
+            }
+            .navgiation {
+                display: inline;
+            }
+            .nav-list-item {
+                display: inline-block;
+            }
+            a {
+                display: inline;
+            }
+            .social-contact {
+                display: inline;
+            }
+            .col-5 {
+                width: 50%;
+            }
+            .footer-nav {
+                display: inline;
+            }
+        </style>
+        <script>
+            console.log ("Hello world!")
+        </script>
+    </head>
+    <body>
+        <div class="container">
+            <div class="row">
+                <div class="column">
+                    <nav>
+                        <ul class="navigation">
+                            <li class="nav-list-item"><a href="#home">LOGO</a></li>
+                            <li class="nav-list-item"><a href="#products">PRODUCTS</a></li>
+                            <li class="nav-list-item"><a href="#solutions">SOLUTIONS</a></li>
+                            <li class="nav-list-item"><a href="#services">SERVICES</a></li>                            
+                            <li class="nav-list-item"><a href="#industries">INDUSTRIES</a></li>
+                            <li class="nav-list-item"><a href="#partners">PARTNERS</a></li>
+                            <li class="nav-list-item"><a href="#support">SUPPORT</a></li>
+                            <li class="nav-list-item"><a href="#blog">BLOG</a></li>
+                            <li class="nav-list-item"><a href="#search">MAGNIFYING GLASS</a></li>
+                            <li class="nav-list-item"><a href="#headphone">HEADPHONE ICON</a></li>
+                        </ul>                                                                        
+                    </nav>
+                </div>
+            </div>
+            <div class="row">
+                <div class="column">
+                    <img src="/images/photo-1.jpg" alt="dark blue background with dots and code">
+                    <h1>BlackBerry 20021 Threat Report Now Available</h1>
+                    <p>Learn about the most significant cybersecurity events of 2020 so you can prepare for the year ahead</p>
+                    <button type="submit">READ THE REPORT</button>
```

Comment:
You usually use type="submit" when you have a form.
This property lets the browser know that when you click it, the form should be submitted.
However, in this case above, there is no form.  
In some cases you would use an anchor tag, styled to look like a button.
Please remove type="submit"
In other cases, a button is perfectly fine, like the above.